apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.ts

```diff
@@ -87,11 +87,6 @@ router.post(
         origHash,
       });
 
-      if (!editor.shouldEdit()) {
-        res.redirect(req.originalUrl);
-        return;
-      }
-
```

Comment:
Behavior change: on this and other pages, we'd previously abort the entire write if this check failed. Now, we'll proceed with it, which means we'll end up pulling in changes from the remote where previously we wouldn't. I think this is fine, but let me know if I missed something.

Reply:
I agree this is fine.